src/libyang/patch/libyang_mgmt_framework.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/tree_data.c b/src/tree_data.c
+index 04653a46..65dca211 100644
+--- a/src/tree_data.c
++++ b/src/tree_data.c
+@@ -842,7 +842,7 @@ error:
+     return ret;
+ }
+ 
+-int
++API int
+ lyd_check_mandatory_tree(struct lyd_node *root, struct ly_ctx *ctx, const struct lys_module **modules, int mod_count,
```

Comment:
can this go to upstream?

Reply:
Yes, this patch is just to export lyd_check_mandatory_tree() as API so that CVL can validate mandatory YANG nodes.

Reply:
Yeah good to upstream with reason of use. Though no harm in making it an API in our code till then.